pkgs/applications/virtualization/lima/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, qemu
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "lima";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "AkihiroSuda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-vwAxVBy2SqghxJGscSZ1o1B8EMvQh1fz5CS1YG7Rq2g=";
+  };
+
+  vendorSha256 = "sha256-xM9LLh5c5QBrcIptdqiNNp1nU9GcdQvwrCnnyuXWYfE=";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  buildPhase = ''
+    runHook preBuild
+    make "VERSION=v${version}" binaries
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -r _output/* $out
+    wrapProgram $out/bin/limactl \
+      --prefix PATH : ${lib.makeBinPath [ qemu ]}
+    installShellCompletion --cmd limactl \
+      --bash <($out/bin/limactl completion bash)
+    runHook postInstall
+  '';
+
+  installPhaseCheck = ''
+    $out/bin/limactl prune 2>&1
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/AkihiroSuda/lima";
+    description = "Linux virtual machines (on macOS, in most cases)";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ anhduy ];
+    platforms = with platforms; linux ++ darwin;
```

Not needed, it's the same as the default set by `buildGoModule`. 
```suggestion
    platforms = with platforms; linux ++ darwin;
```

---

got it, thanks